### PR TITLE
update icon: twitter (original)

### PR DESCRIPTION
**Pull request by Yashh56**
### New Twitter icon was added in SVG form          #1890 
altnames was also added in devicon.json

#1909

### CODE REVIEW

**Review by Snailedlt — CHANGES_REQUESTED** [review]
This is still based on the master branch it seems. Notice the conflicts
image

and the updated workflows file:
image

**Comment by Yashh56** [comment]
How should I resolve this ? I change the base branch to develop

**Comment by Yashh56** [comment]
update altname of twitter (original, original-wordmark) by Yashh56 · Pull Request #1912 · devicons_devicon - Google Chrome 10_19_2023 8_39_10 PM
update altname of twitter (original, original-wordmark) by Yashh56 · Pull Request #1912 · devicons_devicon - Google Chrome 10_19_2023 8_39_44 PM

**Comment by Snailedlt** [comment]
You need to either:
1. Remove the commit that's from the master branch
2. Make a new branch based on the develop branch... That is, checkout the develop branch and create a new branch from there: `git checkout develop && git pull && git checkout -b your_branch_name`

**Comment by Yashh56** [comment]
@Snailedlt  review this PR

**Review by Snailedlt — DISMISSED** [review]
Looking good! Thank you for your contribution! 💯

**Comment by Yashh56** [comment]
when will the PR get merged ? @Snailedlt

**Comment by Snailedlt** [comment]
@Yashh56 when it has at least 2 approvals from the maintainers :)

**Comment by Snailedlt** [comment]
Looks like the PR still has merge conflicts though

**Comment by Yashh56** [comment]
How should I resolve this I want to merge atleast 4 PR till this October end for hacktoberfest :)

**Comment by Yashh56** [comment]
Hey Maintainer Please Review this PR :)

**Review by Snailedlt — APPROVED** [review]

**Comment by Snailedlt** [comment]
Fixed it for you. This PR should count towards your octoberfest now :)

**Comment by Yashh56** [comment]
Thanks @Snailedlt   :)

**Comment by Yashh56** [comment]
What are these commits which you made ? Is my commit are incompletes ?

**Review by EnisMulic — APPROVED** [review]

**Comment by Snailedlt** [comment]
Everything is fine, just the PR title that needed to change a bit so that the checks go through. Should be fine now :)

File: icons/twitter/twitter-original.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#1d9bf0" d="M114.896 37.888c.078 1.129.078 2.257.078 3.396 0 34.7-26.417 74.72-74.72 74.72v-.02A74.343 74.343 0 0 1 0 104.21c2.075.25 4.16.375 6.25.38a52.732 52.732 0 0 0 32.615-11.263A26.294 26.294 0 0 1 14.331 75.09c3.937.76 7.993.603 11.857-.453-12.252-2.475-21.066-13.239-21.066-25.74v-.333a26.094 26.094 0 0 0 11.919 3.287C5.5 44.139 1.945 28.788 8.913 16.787a74.535 74.535 0 0 0 54.122 27.435 26.277 26.277 0 0 1 7.598-25.09c10.577-9.943 27.212-9.433 37.154 1.139a52.696 52.696 0 0 0 16.677-6.376A26.359 26.359 0 0 1 112.92 28.42 52.227 52.227 0 0 0 128 24.285a53.35 53.35 0 0 1-13.104 13.603z"/></svg>
+<svg width="128" height="128" viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M75.916 54.2 122.542 0h-11.05L71.008 47.06 38.672 0H1.376l48.898 71.164L1.376 128h11.05L55.18 78.303 89.328 128h37.296L75.913 54.2ZM60.782 71.79l-4.955-7.086-39.42-56.386h16.972L65.19 53.824l4.954 7.086 41.353 59.15h-16.97L60.782 71.793Z" fill="#000" style="stroke-width:.104373"/></svg>
```
Comment (weh) [inline icons/twitter/twitter-original.svg:1]: `width` and `height` can be removed
```suggestion
<svg viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M75.916 54.2 122.542 0h-11.05L71.008 47.06 38.672 0H1.376l48.898 71.164L1.376 128h11.05L55.18 78.303 89.328 128h37.296L75.913 54.2ZM60.782 71.79l-4.955-7.086-39.42-56.386h16.972L65.19 53.824l4.954 7.086 41.353 59.15h-16.97L60.782 71.793Z" fill="#000" style="stroke-width:.104373"/></svg>

```

**Review by weh — CHANGES_REQUESTED** [review]
Just a really minimal change. Thx :+1:

**Review by weh — APPROVED** [review]
:+1:

File: icons/twitter/twitter-original.svg [hunk 2]
```diff
@@ -1 +1,2 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#1d9bf0" d="M114.896 37.888c.078 1.129.078 2.257.078 3.396 0 34.7-26.417 74.72-74.72 74.72v-.02A74.343 74.343 0 0 1 0 104.21c2.075.25 4.16.375 6.25.38a52.732 52.732 0 0 0 32.615-11.263A26.294 26.294 0 0 1 14.331 75.09c3.937.76 7.993.603 11.857-.453-12.252-2.475-21.066-13.239-21.066-25.74v-.333a26.094 26.094 0 0 0 11.919 3.287C5.5 44.139 1.945 28.788 8.913 16.787a74.535 74.535 0 0 0 54.122 27.435 26.277 26.277 0 0 1 7.598-25.09c10.577-9.943 27.212-9.433 37.154 1.139a52.696 52.696 0 0 0 16.677-6.376A26.359 26.359 0 0 1 112.92 28.42 52.227 52.227 0 0 0 128 24.285a53.35 53.35 0 0 1-13.104 13.603z"/></svg>
+<svg viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M75.916 54.2 122.542 0h-11.05L71.008 47.06 38.672 0H1.376l48.898 71.164L1.376 128h11.05L55.18 78.303 89.328 128h37.296L75.913 54.2ZM60.782 71.79l-4.955-7.086-39.42-56.386h16.972L65.19 53.824l4.954 7.086 41.353 59.15h-16.97L60.782 71.793Z" fill="#000" style="stroke-width:.104373"/></svg>
+
```
Comment (Snailedlt) [inline icons/twitter/twitter-original.svg:2]: Remove extra newline (there were two here)
```suggestion
```

**Pull request merged**